tasks/task2/README.md

```diff
@@ -1,25 +1,63 @@
-# Store
-Let's imagine that we are building the store app.
-It should be the simple store where we can just manage orders and products (as admins).
-Customers should be able to place the order.
+# Store App Task
+
+## Overview
+
+Welcome to the Store App task!
+In this exercise, you will be developing a simple store application where admins can manage products and orders, while customers can place orders.
+Your goal is to make the store functional by following the instructions provided.
 
 Run the tests like before:
+
 ```bash
 npm test
 ```
+
 or
+
 ```bash
 npm run test-watch
 ```
 
-
-Make them green.
+Make sure all tests pass before submitting your solution.
 
 1. Add more tests and code to make the store functional.
 2. Create products repository.
 3. Add shopper and admin user. Only admin can add products.
-4. Calculate shipping cost for placed order:
-  - shipping cost = 3 * quantity + 2
-  - for orders with quantity above 10 shipping cost is always 35
-5. Add a feature where shipping cost is free next Christmas Eve (24.12)
-6. Fix Result type so it is more type safe. Get rid of `any` in details
+
+## Task Instructions
+
+1. Make Tests Pass:
+   Start by making the existing tests pass. You will find these tests in the `e2e.test.ts file`.
+
+2. Add More Tests and Code:
+   Extend the current code in the `/store/index.ts` by adding more functionalities along with corresponding tests to make the store functional.
+
+3. Create Products Repository:
+   Implement a products repository that will handle storing and retrieving product information.
+
+4. Add User Roles:
+   Implement shopper and admin roles. Only admins should have the ability to add products. Customers should be able to place an order.
+
```

Comment:
I guess we can remove this task or suggest candidate how to solve this in the smart way